### DISCUSSION TITLE
Setting MX and TTL record for custom domain

### DISCUSSION THREAD

**Discussion by justcharlz** [vercel/vercel#4671] [Help] [answered]
Hello guys,

How long does it take for MX and TTL record to reflect. I added mine using the UI and uptill now, it is not yet reflecting. I can't receive mail. Any help will be appreciated.

Regards

**Comment by mcsdevv** [comment, marked as answer]
Hello Charlie,

This should only take a few minutes, if that's not the case please contact us using the [support form](https://vercel.com/support/request).

**Reply by justcharlz** [reply]
Thank you. It is working now. I included the full domain name. I think in the Dashboard, users should be told not to include it just like the instructions when using the CLI.
